eras/conway/impl/test/Test/Cardano/Ledger/Conway/BinarySpec.hs

```diff
@@ -28,4 +30,9 @@ spec = do
     roundTripCborSpec @(Voter StandardCrypto)
     roundTripConwayCommonSpec @Conway
     -- ConwayGenesis only makes sense in Conway era
-    roundTripEraSpec @Conway @(ConwayGenesis StandardCrypto)
+    context "Genesis" $ do
+      roundTripCborSpec @(UpgradeConwayPParams Identity)
```

Comment:
This type is tested as part of the ConwayGenesis serialization. It is not meant to be serialized as a standalone type, and the only reason it even has the Enc/DecCBOR instances defined is because it is used in ConwayGenesis.

So, this test is also not needed. And if it was needed, it would have been redundant to use `roundTripCborSpec`, because `UpgradeConwayPParams` can only be used in Conway, but `roundTripCborSpec` would test it for all protocol versions.

Reply:
Got it. Yes, this one needs to be removed. I was testing the parts since the `ConwayGenesis` type was failing. :raised_hands: